tools/generate-files

```diff
@@ -12,8 +12,17 @@ set -e
 
 ROOT="$PWD"
 PREFIX="${PREFIX:-$ROOT/build/local}"
+if [ ! -d $PREFIX ]
```

Comment:
👍we did set prefix in CI

Reply:
From Dockerfile yes, but not if you start a docker image, and run the generate script.